src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -50,7 +51,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
-import java.util.function.Consumer;
+import java.util.function.BiConsumer;
```

Comment:
This might look weird but keep on reading. I have a comment below explaining why we need to switch it to a biconsumer